src/main/java/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package service.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import service.CsvFileWriterService;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    @Override
+    public void writeToNewCsvFile(String fileName, List<String> dataToWrite) {
+        if (fileName == null
+                || dataToWrite == null
+                || !fileName.endsWith(".csv")) {
+            throw new RuntimeException("Incorrect file name, extension or data!");
+        }
+        File file = new File(fileName);
+        StringBuilder builder = new StringBuilder();
+
+        for (String line : dataToWrite) {
+            builder.append(line).append(System.lineSeparator());
+        }
+        try {
+            Files.write(file.toPath(), builder.toString().getBytes(), StandardOpenOption.CREATE);
```

Comment:
there is more suitable method in Files class - you can delete this loop
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/c241a301-b789-4a2a-a8ab-c14b2b73a392)